src/main/java/appeng/core/features/DamagedItemDefinition.java

```diff
@@ -85,33 +83,7 @@ public boolean isSameAs( final ItemStack comparableStack )
 			return false;
 		}
 
-		return this.isEnabled() && comparableStack.getItem() == this.source.get().getItem() && comparableStack.getItemDamage() == this.source.get().getDamage();
+		return this.source.isPresent() && comparableStack.getItem() == this.source.get().getItem() && comparableStack.getItemDamage() == this.source.get().getDamage();
```

Comment:
This should keep `isEnabled()`. While it is currently the same call, it is not really the same semantic.

I cannot remember the actual reason currently, but there was some discussion about changing the definitions in general once we had the change without breaking every addon during 1.7.10. Maybe it was only return the definition as Optional instead of all the `maybe(Item|ItemStack|Block)` (except should I have forgotten some cases where the block and item can be disabled separately)